Dockerfile

```diff
@@ -36,10 +36,10 @@ RUN apt-get update && apt-get install -y \
 RUN pip3 install unittest-xml-reporting
 
 # For git-lfs
-RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
-# Pin version to 2.5.* until a 2.6.* version with a fix
-# for https://github.com/git-lfs/git-lfs/issues/3358 is released.
-RUN apt-get install -y git-lfs=2.5.*
+# git-lfs is not available per default for Ubuntu 16.04
```

Comment:
this is slightly confusing: what do you by "per default"? we're pinning version, so I guess, it's not that default after all?

Reply:
Sure. I can rephrase this.

Reply:
I reworded a bit. Is this better?